controlplane/pkg/common/context.go

```diff
@@ -0,0 +1,271 @@
+// Copyright (c) 2019 Cisco and/or its affiliates.
```

Comment:
Why is this being moved to pkg?

Reply:
It is new common file for local and remote cases.